webapp/channels/src/utils/notifications.ts

```diff
@@ -111,3 +111,20 @@ export function showNotification(
         };
     };
 }
+
+export function isNotificationAPISupported() {
```

Comment:
Could you change `showNotification` above to use these functions so that we always access `Notification` consistently through them?